cogs/deepfry.py

```diff
@@ -0,0 +1,84 @@
+import os
+from tempfile import TemporaryFile
+
+import requests
+
+import discord
+from discord.ext import commands
+from PIL import Image, ImageOps, ImageEnhance
+
+from cogs.base import BaseCog
+
+
+class Colours:
+    RED = (254, 0, 2)
+    YELLOW = (255, 255, 15)
+
+
+class Deepfry(BaseCog):
+    file_types = [
+        'bmp',
+        'gif',
+        'jpg',
+        'jpeg',
+        'png'
+    ]
+
+    @commands.command()
+    async def deepfry(self, ctx):
+        for attachment in ctx.message.attachments:
+            if check_file_type(attachment.filename, self.file_types):
+                img = fry_to_shits(attachment.url)
+                img.save('tmp/deepfry.jpg')
+                await ctx.send(file=discord.File('tmp/deepfry.jpg'))
+                os.remove('tmp/deepfry.jpg')
```

Comment:
fry_to_shits should return a TemporaryFile. you can pass its path to the discord.File